modules/templates/helper.go

```diff
@@ -620,6 +622,16 @@ func Str2html(raw string) template.HTML {
 	return template.HTML(markup.Sanitize(raw))
 }
 
+// Md2html render Markdown text to HTML (non-sanitize)
+func Md2html(raw string) template.HTML {
+	var err error
+	var RenderedContent string
+	if RenderedContent, err = markdown.RenderString(&markup.RenderContext{}, raw); err != nil {
+		return template.HTML(err.Error())
```

Comment:
Here we should return the escaped HTML content. Otherwise users won't read the correct description if it is an invalid Markdown content.

And we need log.Warning to report errors in logs.

Reply:
this should be fixed now